vagrant-pxe-harvester/setup_harvester.sh

```diff
@@ -2,9 +2,28 @@
 
 MYNAME=$0
 ROOTDIR=$(dirname $(readlink -e $MYNAME))
+REPROVISION=False
+SETTINGS_FILE=settings.yml
+PREV_SETTINGS_FILE=.settings.yml.orig
```

Comment:
.settings.yml.orig is copied from settings.yml after the first setup_harvester.sh.
Then we could compare them next time setup_harvester.sh

If .setting.yml.org does not exist, `diff` return value is 2 (ENOENT).
The ENOENT would not trigger provision because we assume that is the first setup_harvester.sh.

Ah, the comment does not correct, I will update it.